homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1026,11 +1026,16 @@ async def prepare_text_to_speech(self) -> None:
             tts_options[tts.ATTR_VOICE] = self.pipeline.tts_voice
 
         if self.tts_audio_output is not None:
-            tts_options[tts.ATTR_PREFERRED_FORMAT] = self.tts_audio_output
-            if self.tts_audio_output == "wav":
-                # 16 Khz, 16-bit mono
-                tts_options[tts.ATTR_PREFERRED_SAMPLE_RATE] = 16000
-                tts_options[tts.ATTR_PREFERRED_SAMPLE_CHANNELS] = 1
+            if isinstance(self.tts_audio_output, str):
+                # Audio format only
+                tts_options[tts.ATTR_PREFERRED_FORMAT] = self.tts_audio_output
+                if self.tts_audio_output == "wav":
+                    # 16 Khz, 16-bit mono
+                    tts_options[tts.ATTR_PREFERRED_SAMPLE_RATE] = 16000
+                    tts_options[tts.ATTR_PREFERRED_SAMPLE_CHANNELS] = 1
+            else:
+                # Merge options that were passed in
+                tts_options.update(self.tts_audio_output)
```

Comment:
Should we blow up if unsupported options are sent in? Or should we log a warning? 

Also, this line is the thing that's new, and it's not tested.